lib/sdf-server/src/server/service/pkg/install_pkg.rs

```diff
@@ -86,8 +85,8 @@ pub async fn install_pkg(
 async fn install_pkg_inner(
```

Comment:
This can happen in a separate pass (no need to block the PR) - but would be good to pull the module nomenclature into these endpoints

Reply:
yeah my plan was to rename these to *_module ;) That PR will follow shortly!